### PR TITLE
updated readme

### LINKED ISSUE #154: add documentation
update readme for project description. Also add documentation based on project theme.

**would you like to work on this?**
yes

### PR DESCRIPTION
**Pull request by DravitLochan**
Fixes #154 updated readme accordingly.

### CODE REVIEW

**Comment by mariobehling** [comment]
Fix spelling errors!

**Comment by DravitLochan** [comment]
@mariobehling done!

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
...
 
-Susi is an artificial intelligence combining pattern matching, internet data, data flow principles and inference engine principles. It will have some reflection abilities and it will be able to remember the users input to produce deductions and a personalized feed-back. Its purpose is to explore the abilities of an artificial companion and to answer the remaining unanswered questions. The accounts.susi.ai repo is a front-end that is developed for managing user accounts of susi.
+Susi is an artificial intelligence combining pattern matching, internet data, data flow principles and inference engine principles. It will have some reflection abilities and it will be able to remember the users input to produce deductions and a personalized feed-back. Its purpose is to explore the abilities of an artificial companion and to answer the remaining unanswered questions. The accounts.susi.ai repo is a front-end web application that is developed for managing user accounts of susi. The project is hosted on gh-pages at http://accounts.susi.ai.
```
Comment (dynamitechetan) [inline README.md]: please change feed-back to feedback

**Comment by mariobehling** [comment]
Please follow best practices.

**Pull request merged**